test/lit.cfg

```diff
@@ -1593,19 +1593,13 @@ elif (run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'openbsd', 'windows-
         config.import_libdispatch = ('-I %s -I %s -L %s'
             % (libdispatch_source_dir, libdispatch_swift_module_dir, libdispatch_artifact_dir))
 
-    libdispatch_static_artifact_dir = config.libdispatch_static_build_path
-    libdispatch_swift_static_module_dir = make_path(libdispatch_static_artifact_dir, 'src', 'swift', 'swift')
+    libdispatch_static_artifact_dir = os.path.join(config.libdispatch_static_build_path, 'lib')
     libdispatch_static_artifacts = [
-        make_path(libdispatch_static_artifact_dir, 'src', 'libdispatch.a'),
-        make_path(libdispatch_static_artifact_dir, 'src', 'swift', 'libswiftDispatch.a'),
-        make_path(libdispatch_swift_static_module_dir, 'Dispatch.swiftmodule')]
+        make_path(libdispatch_static_artifact_dir, 'libdispatch.a'),
+        make_path(libdispatch_static_artifact_dir, 'libBlocksRuntime.a')]
     if (all(os.path.exists(p) for p in libdispatch_static_artifacts)):
         config.available_features.add('libdispatch_static')
-        config.import_libdispatch_static = ('-I %s -I %s -L %s -L %s -L %s'
-            % (libdispatch_source_dir, libdispatch_swift_static_module_dir,
-            make_path(libdispatch_static_artifact_dir, 'src'),
-            make_path(libdispatch_static_artifact_dir, 'src', 'BlocksRuntime'),
-            make_path(libdispatch_static_artifact_dir, 'src', 'swift')))
+        config.import_libdispatch_static = '-L %s' % libdispatch_static_artifact_dir
```

Comment:
Since neither the overlay nor the headers are used for static linking, this is simplified to a single library search path flag.